hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1034,6 +1035,24 @@ public function upgrade_1025() {
   }
 
   /**
+   * Alters civicrm_hrjobcontract_hour table to make default values for fte_num
+   * and fte_denom equal to 0.
+   * 
+   * @return boolean
+   *   True on success
+   * @todo Rename upgrade!!  To upgrade_1026()  Include in upgrade bundle!
+   */
+  public function upgrade_1026() {
+    $query = '
```

Comment:
Just to make this a bit cleaner, use double quotes around the string and then you can remove the escape chars inside the query